test/org/javarosa/form/api/AttributesTestCase.java

```diff
@@ -0,0 +1,68 @@
+package org.javarosa.form.api;
```

Comment:
How did you choose the package to put this in? It feels more like it would belong in `xform.parse` to me.